### ISSUE TITLE
OpenCpu docs for Quectel BC66

### ISSUE THREAD

**Issue by ASL07** [Wiz-IO/platform-opencpu#2]
Hi,
Thanks for the great work!
I am trying to find the documentation and source code OpenCPU SDK for BC66
I am trying out the Quectel BC66 module from Olimex, and would like to compile without PlatformIO and without Arduino
Could you point me to the right direction?

**Comment by Wiz-IO** [comment]
Hi,
I am not authorized to share Quectel documentations... etc
Contact with your local Quectel support or
https://www.quectel.com/support/contact.htm

Olimex: support@olimex.com

**Comment by ASL07** [comment]
I have already done it, waiting on their response
I thought there may be public docs
Thank you

**Comment by Wiz-IO** [comment]
BTW: PIO is just IDE
as default my port follow the QuecOpen SDKs ( as is )
for PIO I use "standart" GCC-none-eabi, and not have problems with compiler
The builder is python editable scripts .... ALL can be edit from the user

If you want to create OpenCPU project - just select this from New Project Wizard and follow the Quectel SDK, compile and upload...
